scripts/background.js

```diff
@@ -286,86 +342,36 @@ chrome.webRequest.onCompleted.addListener(function(details) {
                     //If show-all Context is true, Create a window which is focussed and create tabs in it
                     if(event2.showall==true){
                       chrome.windows.create({url:chrome.runtime.getURL("doi.html")+"?url="+message.url, width:800, height:800, top:0, left:0, focused:true},function(win){
-                        windowIdtest=win.id;
-                        chrome.windows.onRemoved.addListener(function (win1) {
```

Comment:
Please add chrome.windows.onRemoved.addListener event for `windowIdtest` in line No- 68

Reply:
Please add `var alexa_url="https://archive.org/services/context/alexa?url="+url;` in line number 448. Thank you.

Reply:
You can view the code in github "Files changed", click on a specific line and add a comment there.